spec/factories/kudos.rb

```diff
@@ -25,6 +28,7 @@
     title { 'Kudo-test' }
     content { 'Don\'t mind me, just testing!' }
 
+    company_value factory: :company_value
```

Comment:
if the name of the factory is the same as name of association, then you can simply write:
```suggestion
    company_value 
```